### PR TITLE
#143 CallableWithFallback

### PR DESCRIPTION
**Pull request by cheparinV**
As per #143

### CODE REVIEW

File: src/main/java/org/cactoos/func/CallableWithFallback.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * SOFTWARE.
+ */
+package org.cactoos.func;
+
+import java.util.concurrent.Callable;
+import org.cactoos.Proc;
+
+/**
+ * Runnable with a fallback plan.
+ *
```
Comment (alex-semenyuk) [inline src/main/java/org/cactoos/func/CallableWithFallback.java]: @cheparinV Typo "Callable"

File: src/main/java/org/cactoos/func/CallableWithFallback.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    /**
+     * The callable.
+     */
+    private final Callable<Y> callable;
+
+    /**
+     * The fallback.
+     */
+    private final Proc<Throwable> fallback;
```
Comment (yegor256) [inline src/main/java/org/cactoos/func/CallableWithFallback.java]: @cheparinV why not `Func<Throwable, Y>`?

**Comment by yegor256** [comment]
@cheparinV I would use `Func` instead, see my comment above

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**